misc/config_tools/configurator/packages/configurator/src/pages/Config/Board.vue

```diff
@@ -158,6 +159,38 @@ export default {
             })
       }
     },
+    reformatBoardInfo() {
+      if (this.board.BIOS_INFO != null) {
+        let info = this.board.BIOS_INFO.split('\n')
+        let revindex = 0, infoindex = 0
+        for (const tempElement of info) {
```

Comment:
If `info` is an array, you can use something like below to get the index for each iteration so that you do not need to get the index by another `indexOf` search.

```
for (let [index, element] of info.entries()) {
    /* ... */
}
```